components/draw/GeometryMode.svelte

```diff
@@ -135,11 +135,11 @@
 
     currentlyEditing = id;
 
-    if (feature.geometry.type == "LineString") {
+    if (feature!.geometry.type == "LineString") {
```

Comment:
`!` means assert that `feature` isn't `undefined`. This is valid because we only call `startEditing` with IDs of stuff that exists